website/content/usage/_index.md

```diff
@@ -20,6 +20,12 @@ address, or if the address is already in use by another service,
 assignment will fail and MetalLB will log a warning event visible in
 `kubectl describe service <service name>`.
 
+We both support `spec.loadBalancerIP` and a custom `metallb.universe.tf/loadBalancerIPs`
```

Comment:
metalLB supports `spec.loadBalancerIP` and a custom `metallb.universe.tf/loadBalancerIPs`

Reply:
done